docs/memory_profiler.md

```diff
@@ -0,0 +1,10 @@
+## Enable Memory Profiling
+
+Launch training job with the following command (or alternatively set configs in toml files)
+```
+CONFIG_FILE="./train_configs/debug_model.toml" ./run_llama_train.sh --profiling.enable_memory_snapshot --profiling.save_memory_snapshot_folder output_folder
+```
+* `--profiling.enable_memory_snapshot`: enable memory snapshot
+* `--profiling.save_memory_snapshot_folder`: dump memory snapshots in to output folder, default to be `memory_snapshot`.
```

Comment:
The default here should be `./outputs/memory_snapshot`

Reply:
I mean the default value for this field is `memory_snapshot`, but you should mention it would be under the output folder, which is controlled by `job.dump_folder`.